src/components/views/settings/tabs/user/SecurityUserSettingsTab.tsx

```diff
@@ -372,7 +375,19 @@ export default class SecurityUserSettingsTab extends React.Component<IProps, ISt
                     { crossSigning }
                     <CryptographyPanel />
                 </div>
-                { privacySection }
+                <div className="mx_SettingsTab_heading">{ _t("Privacy") }</div>
+                <div className="mx_SettingsTab_section">
+                    { analytics }
+                    <span className="mx_SettingsTab_subheading">{ _t("Presence") }</span>
```

Comment:
This heading is confusing - even to users "presence" means online/offline status and not necessarily typing notifications and such.

Reply:
Hmm, I've always interpreted the word as  something that indicates the user's _presence_ on the other side, both RRs and typing notifs would fall under that

That said, I might be just weird in the way I interpreted that - do you have a suggestion for a better heading or should I wait for what design suggests?

Reply:
other words would be things like "metadata", but might be overly alarmist for the app. Design is definitely a lot better at these sorts of things :)